batch_job/src/Makefile

```diff
@@ -3,7 +3,7 @@ include ../../rules.mk
 
 LIBRARIES = libbatch_job.a
 
-PROGRAMS = work_queue_factory work_queue_pool
+PROGRAMS = work_queue_factory work_queue_pool amazon_batch_cleanup amazon_batch_initialize
```

Comment:
A small thing, but important for keeping our namespace clean:
Call these scripts `makeflow_amazon_batch_setup` and `makeflow_amazon_batch_cleanup`